pkg/utils/tp_proc_test.go

```diff
@@ -16,6 +16,17 @@ var (
 	testTPDepositSpent  float64 = 32.64787
 )
 
+func TestCalcTPOrderError(t *testing.T) {
```

Comment:
Usually you test for behavior of a specific error: invalid arguments or behavior if one of the dependency methods returns error